stories/modules-tagmanager-settings.stories.js

```diff
@@ -118,20 +118,29 @@ storiesOf( 'Tag Manager Module/Settings', module )
 		registry.dispatch( STORE_NAME ).setInternalContainerID( container.containerId );
 		registry.dispatch( STORE_NAME ).receiveGetSettings( defaultSettings );
 
-		return <Settings isOpen={ true } isEditing={ true } registry={ registry } />;
+		return <Settings isOpen isEditing registry={ registry } />;
 	} )
 	.add( 'Edit, with no accounts', ( registry ) => {
 		registry.dispatch( STORE_NAME ).receiveGetAccounts( [] );
 		registry.dispatch( STORE_NAME ).receiveGetSettings( defaultSettings );
 
-		return <Settings isOpen={ true } isEditing={ true } registry={ registry } />;
+		return <Settings isOpen isEditing registry={ registry } />;
 	} )
 	.add( 'Edit, with "Set up a new account"', ( registry ) => {
 		registry.dispatch( STORE_NAME ).setAccountID( ACCOUNT_CREATE );
 		registry.dispatch( STORE_NAME ).receiveGetAccounts( fixtures.accounts );
 		registry.dispatch( STORE_NAME ).receiveGetSettings( defaultSettings );
 
-		return <Settings isOpen={ true } isEditing={ true } registry={ registry } />;
+		return <Settings isOpen isEditing registry={ registry } />;
+	} )
+	.add( 'Edit, with "Set up a new container"', ( registry ) => {
```

Comment:
Please also add a case to show the error/warning state when a container by the same name exists.

Reply:
Added.